manifests/init.pp

```diff
@@ -0,0 +1,166 @@
+# Sets up rkhunter to be run once per day
+#
+# @param minute_upd   Cron minute for update check
+# @param hour_upd     Cron hour for update check
+# @param monthday_upd Cron monthday for update check
+# @param month_upd    Cron month for update check
+# @param weekday_upd  Cron weekday for update check
+# @param minute       Cron minute
+# @param hour         Cron hour
+# @param monthday     Cron monthday
+# @param month        Cron month
+# @param weekday      Cron weekday
+#
+# @param package_ensure                The ensure status of packages to be managed
+#
+# @param rkhunter_conf_file            The path to the conf file
+#
+# @param rkhunter_conf_file_template   Template to use for the conf file
+#
+# @param check_for_updates             Check internet for definition updates
+#
+# @author https://github.com/simp/pupmod-simp-rkhunter/graphs/contributors
+#
+class rkhunter (
```

Comment:
I would definitely break this up into at least a `rkhunter::config` class since that lets you separate your configuration items away from your code flow control items.